internal/cmd/manager/controller/controller.go

```diff
@@ -314,6 +307,20 @@ func RunController(
 	return nil
 }
 
+func getNamespacesToWatch(conf *configuration.Data) map[string]cache.Config {
+	if len(conf.WatchNamespace) == 0 {
+		setupLog.Info("Listening for changes on all namespaces")
+		return nil
+	}
+	namespaces := map[string]cache.Config{}
+	for _, namespace := range conf.WatchedNamespaces() {
+		namespaces[namespace] = cache.Config{}
+	}
+
+	setupLog.Info("Listening for changes", "watchNamespaces", namespaces)
```

Comment:
We'll get an error in the logs this way:
```
{"level":"info","ts":"2024-10-01T12:33:02Z","logger":"setup","msg":"Listening for changes","watchNamespacesError":"json: unsupported type: cache.TransformFunc"}
```

```suggestion
	setupLog.Info("Listening for changes", "watchNamespaces", 
		append(conf.WatchedNamespaces(), conf.OperatorNamespace))

```